src/main/webapp/index.jsp

```diff
@@ -6,10 +6,10 @@
       <div class="home-page content">
         <h1 class="welcome">Welcome to Piece of Cake!</h1>
         <form class="searchbar" method="GET" action="search">
-             <input type="text" class="form-control searchbar" placeholder="Type recipe title" name="searchterm">
+             <input type="text" class="form-control searchbar" placeholder="Recipe title or ingredients separated by comma" name="searchterm">
```

Comment:
Please check the comment I've left before in this same section and act on it accordingly.